keyboard.html

```diff
@@ -0,0 +1,97 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <title>||Working with elements||</title>
+  <style>
+    #keyboard {
+      display: flex;
+    }
+    .white-key {
+      height: 100px;
+      width: 28px;
+      background-color: white;
+      outline-style: solid;
+      outline-width: thin
+    }
+
+    .black-key {
+      height: 60px;
+      background-color: black;
+      position: absolute;
+      outline-style: solid;
+      outline-width: thin
+    }
+  </style>
+</head>
+<body>
+  <div id="description"></div>
+  <div id="keyboard"></div>
+  <script> 
+    const keyboard = document.getElementById('keyboard')
+
+    document.body.onload = addKeys(14);
+    window.addEventListener('resize', resizeAllBlackKeys)
+    const allBlackKeysArray = Array.from(document.getElementsByClassName('black-key'))
+    
+    function addKeys(numKeys) {
+      const whiteKeysPerOctave = 6
+      let i = 0;
+      let whiteKeyCount = 0;
+      while(i < numKeys) {
+        // create a new rectangle div element
+        const newDiv = document.createElement("div");
+        newDiv.setAttribute('id', `white-key${i}`)
+        newDiv.classList.add('white-key')
+    
+        // add the newly created element and its content into the DOM
+        keyboard.appendChild(newDiv);
+
+        var coords = getPos(document.querySelector(`#white-key${i}`));
```

Comment:
also dont think youre using this, but you could based on my other comment

Reply:
oh wow amazing